packages/jsii/test/test.docs.ts

```diff
@@ -269,6 +269,28 @@ export = {
     test.done();
   },
 
+  // ----------------------------------------------------------------------
+
+  async 'can mark @imported'(test: Test) {
+    const assembly = await compile(`
+      /**
+       * @imported
+       */
+      export class Foo {
+        public floop() {
+          Array.isArray(3);
+        }
+      }
+    `);
+
+    const classType = assembly.types!['testpkg.Foo'] as spec.ClassType;
+    const method = classType.methods!.find(m => m.name === 'floop');
+
+    test.deepEqual(classType.docs!.stability, spec.Stability.Imported);
+    test.deepEqual(method!.docs!.stability, spec.Stability.Imported);
+    test.done();
```

Comment:
Don’t we need a test which verifies that imported can break with only warnings?

Reply:
Yep: https://github.com/aws/jsii/pull/596/files#diff-bb49180729f436b791f8064a35fd60d4R24